modules/op-running-as-root-user-namespace.adoc

```diff
@@ -0,0 +1,34 @@
+[id='op-running-as-root-user-namespace_{context}']
+= Running as root in a user namespace
+
+When you run <something> in a user namespace, your container runs as root (user id 0) but has user privileges on the host. {product-title} uses https://cri-o.io/#:~:text=What%20is%20CRI%2DO%3F,as%20the%20runtime%20for%20kubernetes[CRI-O] as its container runtime interface. CRI-O has a workload feature to run a pod in the user namespace.
+
+To run as a user in a namespace with Openshift Pipelines you need to be able to pass the annotations expected by CRIO to get the pods running  user namespaced. You can directly edit the buildah ClusterTask with `oc edit clustertask buildah` to add this to all users or export the ClusterTask as a task to a specific namespace. 
+
+With the latest `tkn create` feature you can automate it.
+
+[source,terminal,subs="attributes+"]
+----
+$ oc new-project test
+$ tkn task create --from=buildah
+$ oc edit task buildah
+----
```

Comment:
```suggestion
Use `tkn create` feature to automate <finish this phrase>.
+
[source,terminal,subs="attributes+"]
----
$ oc new-project test
$ tkn task create --from=buildah
$ oc edit task buildah
----
```

Reply:
Replace items inside angle brackets, like `<finish this phrase>`, with appropriate values.